File: drools_projects/org.kermeta.smartadapters.drools.diva/src/factory/weaver/Activator.java
package factory.weaver;

import java.util.Properties;

import org.osgi.framework.BundleActivator;
import org.osgi.framework.BundleContext;

import eu.diva.factoryinstdiva.Factory;

public class Activator implements BundleActivator{

	public static BundleContext context;
	
	public void start(BundleContext context) throws Exception {
		Activator.context = context;
		Properties props = new Properties();
		props.put("Factory", "Weaver");
		context.registerService(
				Factory.class.getName(), factory.weaver.Factory.getFact(), props);
	}

	
	public void stop(BundleContext context) throws Exception {

	}

}

File: art2_projects/org.kermeta.art2.runtime/src/main/scala/org/kermeta/art2/runtime/Activator.scala

package org.kermeta.art2.runtime

import org.osgi.framework.BundleContext
import org.osgi.framework.BundleActivator

class Activator extends BundleActivator with fr.irisa.triskell.scala.generated.fw.ImplicitConversion {
	
	def start( context: BundleContext ) {
		handler.actualContext  = context
		handler.init
		var launcher = org.kermeta.artKomparator.RichFactory.createLauncher
		launcher.KersetNodeName("duke.irisa.fr")
		launcher.KersetHotdeployPATH("/Users/ffouquet/Servers/apache-servicemix-4.0.0/deploy")
		launcher.launchFromURL("///Users/ffouquet/Documents/DEV/workspaces/runtime-artIDE/Drop/multi_sample.xmi")
     }   

     def stop( context: BundleContext )  {

     }


}



